src/components/toasts/OfflineToast.js

```diff
@@ -1,15 +1,15 @@
 import lang from 'i18n-js';
 import React from 'react';
-import { web3Provider } from '../../handlers/web3';
 import Toast from './Toast';
 import { useAccountSettings, useInternetStatus } from '@/hooks';
-import { Network } from '@/chains/types';
+import { ChainId } from '@/chains/types';
+import { useConnectedToHardhatStore } from '@/state/connectedToHardhat';
 
 const OfflineToast = () => {
   const isConnected = useInternetStatus();
-  const { network } = useAccountSettings();
-  const providerUrl = web3Provider?.connection?.url;
-  const isMainnet = network === Network.mainnet && !providerUrl?.startsWith('http://');
+  const { chainId } = useAccountSettings();
+  const { connectedToHardhat } = useConnectedToHardhatStore;
```

Comment:
you gotta initialize the hook, `useConnectedToHardhatStore()`

Reply:
responding to your comment re: passing in `chainId` vs `provider` - I should've added a note but I am planning on doing that as a follow up (so I could reduce the amount of places I changed in this PR since most places were passing in the provider) but yes I agree!

Reply:
added a ticket here: https://linear.app/rainbow/issue/APP-1944/replace-loadwallet-param-of-provider-with-chainid